### PR TITLE
Feature/nishant/add speaker detail screen

### PR DESCRIPTION
**Pull request by nisrulz**
Changed the layout for speaker detail a little bit. The icons look out of place so I removed them and put the links in text the way we have in agenda detail. Looks more in line with what we have already.

screenshot_20180309-165513

### CODE REVIEW

File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/Injection.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.mentalmachines.droidcon_boston.utils
+
+import com.google.gson.Gson
+
+class Injection {
```
Comment (kenyee) [inline Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/Injection.kt]: this isn't really injection..it's more a service locator as JW would say ;-)

Comment (nisrulz) [reply Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/Injection.kt]: I can put the icons back in. I just wanted to experiment a little 😅 
About the Injection class, I know it is Service locator, needed to isolate the GSON dependency so named it.

Comment (kenyee) [reply Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/utils/Injection.kt]: experimenting is fair...was just voicing an opinion ;-)

File: Droidcon-Boston/app/src/main/res/layout/speaker_detail_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            android:paddingRight="16dp">
+
+            <TextView
+                android:id="@+id/tv_twitter"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:paddingBottom="8dp"
+                android:paddingEnd="8dp"
+                android:paddingLeft="0dp"
+                android:paddingRight="8dp"
```
Comment (cbonoz) [inline Droidcon-Boston/app/src/main/res/layout/speaker_detail_fragment.xml]: should start using dimens I think for these '8dp', '16dp' values at some point

Comment (nisrulz) [reply Droidcon-Boston/app/src/main/res/layout/speaker_detail_fragment.xml]: 👍 let me make that fix.

**Comment by nisrulz** [comment]
Updated the code to show icons instead of links for social media links
screenshot_20180310-103652

**Comment by kenyee** [comment]
I like the look of the icons fwiw 😀

**Pull request merged**